eclair-core/src/test/scala/fr/acinq/eclair/integration/ChannelIntegrationSpec.scala

```diff
@@ -738,10 +738,9 @@ class AnchorOutputChannelIntegrationSpec extends ChannelIntegrationSpec {
     sender.expectMsg(revokedCommitTx.txid)
     // get the revoked commitment confirmed: now HTLC txs can be published
     generateBlocks(bitcoincli, 2)
-    bitcoinClient.publishTransaction(htlcSuccess.head).pipeTo(sender.ref)
-    sender.expectMsg(htlcSuccess.head.txid)
-    bitcoinClient.publishTransaction(htlcTimeout.head).pipeTo(sender.ref)
-    sender.expectMsg(htlcTimeout.head.txid)
+    // NB: there is a race between C and F here; C may publish more quickly and claim the HTLC outputs directly from the commit tx
```

Comment:
```suggestion
    // NB: The test cannot be deterministic because there is a race between C and F here; C may publish more quickly and claim the HTLC outputs directly from the commit tx. As a result we may have different combinations of transactions if the test is run several times. It's okay, we just need to make sure that the test never fails.
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1564/commits/80c43cfbfe6bfcedb53a4e3f6c3bbcd08434c559